lib/sheet.js

```diff
@@ -37,6 +37,28 @@ const setCell = (target, prop, value) => {
     const options = { context: target.context };
     const script = metavm.createScript(prop, src, options);
     target.expressions.set(prop, script.exports);
+  } else if (prop.includes('.')) {
+    const rootKey = prop.slice(0, prop.indexOf('.'));
+    let targetKey = rootKey;
+
+    const proxyValue = new Proxy(
+      {},
+      {
+        get: (_, nextKey) => {
+          targetKey += `.${nextKey}`;
+
+          if (target.data.has(targetKey)) {
+            const value = target.data.get(targetKey);
+            targetKey = rootKey;
+            return value;
+          }
+          return proxyValue;
```

Comment:
It is not a good idea to make this proxy universal for all names in chain of names. We need new proxy each time we go deeper.